docs/api/selector.md

```diff
@@ -12,6 +12,16 @@ follows:
 - class syntax (`.foo`, `.foo-bar`, etc.)
 - tag syntax (`input`, `div`, `span`, etc.)
 - id syntax (`#foo`, `#foo-bar`, etc.)
+- attr syntax(`[foo="bar"]`, `[foo]`, etc.);
+
+A note on attribute selectors: For compatibilty, you can use either the
+react prop name, or the native html selector. For example, the following
```

Comment:
as I understand it (@lelandrichardson can confirm) enzyme was originally built to replicate much of jQuery's API, which is why the querySelectorAll syntax is used for the selector string. If we're going to be deviating from it, for React Native or otherwise, that's a bigger discussion - does React Native itself use selector syntax?